src/shared/components/annotation/genomeNexus/AlphaMissense.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import annotationStyles from './../styles/annotation.module.scss';
+import classNames from 'classnames';
+import tooltipStyles from './styles/alphaMissenseTooltip.module.scss';
+
+export interface IAlphaMissenseProps {
+    alphaMissensePrediction: string | undefined; // benign, pathogenic, ambiguous
+    alphaMissenseScore: number | undefined;
+}
+
+export function hideArrow(tooltipEl: any) {
+    const arrowEl = tooltipEl.querySelector('.rc-tooltip-arrow');
+    arrowEl.style.display = 'none';
+}
+
+export const AlphaMissense_URL: string =
```

Comment:
Do we need an underscore here?

```suggestion
export const AlphaMissenseURL: string =
```